utils/bundles.stanza

```diff
@@ -29,39 +29,35 @@ public pcb-bundle power :
 public pcb-bundle fpga-diff : 
   diff-pair-pins()
 
-public pcb-bundle jtag :
+public pcb-enum ocdb/bundles/JTAGPins :
+  TRSTN
+
+public pcb-bundle jtag (pins:Tuple<ocdb/bundles/JTAGPins>):
```

Comment:
I believe you only need the fully qualified name in the type definition, this should be ok:

```
public pcb-bundle jtag (pins:Tuple<JTAGPin>):
```